src/assets/CollectiblesController.ts

```diff
@@ -47,11 +47,14 @@ import { compareCollectiblesMetadata } from './assetsUtil';
  * @property externalLink - External link containing additional information
  * @property creator - The collectible owner information object
  * @property isCurrentlyOwned - Boolean indicating whether the address/chainId combination where it's currently stored currently owns this collectible
+ * @property isTransacting - Boolean indicating if the Collectible is being transacting
  */
 export interface Collectible extends CollectibleMetadata {
   tokenId: string;
   address: string;
   isCurrentlyOwned?: boolean;
+  isTransacting?: boolean;
```

Comment:
If we are the ones that set this — and since this is a boolean — then should this be optional?